jetty-servlet/src/main/java/org/eclipse/jetty/servlet/StatisticsServlet.java

```diff
@@ -101,6 +101,12 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
             }
         }
 
+        String statsReset = req.getParameter( "statsReset" );
+        if (statsReset != null && "true".equalsIgnoreCase( statsReset )) {
```

Comment:
ok I will change the other part other code in this class which use this pattern.